apps/src/code-studio/teacherPanelRedux.js

```diff
@@ -75,9 +76,11 @@ const getLevelProgressQueryParams = state => {
       script_id: state.progress.scriptId,
     };
   } else {
+    const currentLevel = getCurrentLevel(state);
+    // If we are on a sublevel, use the level ID of the main (parent) level
     return {
       script_id: state.progress.scriptId,
-      level_id: state.progress.currentLevelId,
+      level_id: currentLevel.parentLevelId || state.progress.currentLevelId,
```

Comment:
The `level_id` value here is used by the `api/teacher_panel_progress` API, which doesn't work with a sublevel ID (because the API tries to look up the associated script level, which doesn't exist for sublevels). The change here retains the original behavior to use the top-level/parent level ID if we're on a sublevel.